src/log.py

```diff
@@ -0,0 +1,61 @@
+import logging
+import logging.handlers
+from pathlib import Path
+
+import coloredlogs  # type: ignore[reportMissingTypeStubs]
```

Comment:
consider using loguru instead, no pressure though as i don't know how it compares to coloredlogs

Reply:
I like using logging with coloredlogs as logging is built into python.